src/data/real/liouville.lean

```diff
@@ -167,4 +166,44 @@ begin
     exact (mem_roots fR0).mpr (is_root.def.mpr hy) }
 end
 
-end irrational
+theorem transcendental {x : ℝ} (liouville_x : liouville x) :
```

Comment:
If the name of the property is `is_transcendental`, then the name of the theorem should also be `is_transcendental`. I think I would just remove the `is_` everywhere.

Reply:
I removed the `is_` in both `transcendental` and `liouville`.  This also allowed one use of dot-notation!